miio/airpurifier.py

```diff
@@ -153,16 +185,22 @@ class AirPurifier(Device):
     def status(self) -> AirPurifierStatus:
         """Retrieve properties."""
 
-        properties = ['power', 'aqi', 'humidity', 'temp_dec',
-                      'mode', 'led', 'led_b', 'buzzer', 'child_lock',
-                      'bright', 'favorite_level', 'filter1_life',
-                      'f1_hour_used', 'use_time', 'motor1_speed']
+        properties = ['power', 'aqi', 'average_aqi', 'humidity', 'temp_dec',
+                      'mode', 'favorite_level', 'filter1_life', 'f1_hour_used',
+                      'use_time', 'motor1_speed', 'purify_volume', 'f1_hour',
+                      # Second request
+                      'led', 'led_b', 'bright', 'buzzer', 'child_lock', ]
 
         values = self.send(
             "get_prop",
-            properties
+            properties[0:13]
```

Comment:
Mind leaving a comment behind to explain why using only a slice here? Same below too.